mautic/mautic
# add background image, background size, and background repeat to `data-section`

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | Yes
| Automated tests included? | No
| Related user documentation PR URL | -
| Related developer documentation PR URL |  -
| Issues addressed (#s or URLs) | -
| BC breaks? | -
| Deprecations? | -

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. 
2. 

#### Steps to test this PR:
1. Open Email Builder, Select The `data-section` element
2. Then you'll see additional field in `Customize Section` menu to change the background image, background size, and background repeat, input some value to it, then save the email
3. Make sure the document is the same after and before you save it, and make sure when you re-click the `data-section` element the value appear in the input field 

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
